client/signup/steps/user/index.jsx

```diff
@@ -100,8 +98,7 @@ export class UserStep extends Component {
 
 	state = {
 		recaptchaClientId: null,
-		experiment: null,
-		isDesktop: isDesktop(),
+		isMobile: isMobile(),
```

Comment:
Do we need to store this info in the component state? What purpose does it serve?